rskj-core/src/test/java/co/rsk/peg/federation/VoteFederationChangeTest.java

```diff
@@ -305,6 +305,19 @@ void voteAddFederatorPublicKey_whenAdding100Federators_returnsSuccessfulResponse
 
 
     // vote commit federation tests
+    @Test
+    void voteCommitFederation_withNullPendingFederation_returnsFederationNonExistentResponseCode() {
+        // Arrange
+        Keccak256 randomHash = Keccak256.ZERO_HASH;
+        ABICallSpec commitFederationAbiCallSpec = new ABICallSpec(FederationChangeFunction.COMMIT.getKey(), new byte[][]{ randomHash.getBytes() });
+
+        // Act
+        int result = federationSupport.voteFederationChange(firstAuthorizedTx, commitFederationAbiCallSpec, signatureCache, bridgeEventLogger);
```

Comment:
```suggestion
        int actualFedChangeResponseCode = federationSupport.voteFederationChange(firstAuthorizedTx, commitFederationAbiCallSpec, signatureCache, bridgeEventLogger);
```

Wdyt? I think `result` is too generic.

Reply:
is called result in all the other tests, no big deal i think